index.md

```diff
@@ -1,9 +1,23 @@
 ---
-# Feel free to add content and custom Front Matter to this file.
-# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-
 layout: default
 ---
 Hello & welcome to Scanly, a university project for module 206CDE. The project is an open-source chatbot, licensed under the [GNU Lesser General Public Licence](/licence). For the task, we have to use [botkit.ai](https://botkit.ai/) & as BSc Cybersecurity & Ethical Hacking students, we have decided to make a chatbox frontend for security tools. The main aims that we have for this project is to make security tools more easier to use for beginers, & hope that this chatbox can be utilised by SMB IT Departments, in order to improve their network security.
```

Comment:
```suggestion
Hello and welcome to Scanly, a university project for module 206CDE. The project is an open-source chatbot, licensed under the [GNU Lesser General Public License](/license). For the task, we have to use [botkit.ai](https://botkit.ai/) and, as Cybersecurity & Ethical Hacking students, we have decided to make a chatbox frontend for security tools. The main aim that we have for this project is to make security tools easier to use for beginners, and we hope that this chatbox can be utilized by SMB IT Departments, in order to improve their network security.
```
May need to rename the license page to fix the spelling

Reply:
I believe I am right & you are looking at Americanised spelling? Can you verify? Source: http://english-grammar-lessons.co.uk/easily_confused/licence_license.htm

Reply:
Huh, didn't know that. Yh, I tend to go with american spelling nowadays considering that everything else uses it like color in configs etc

Reply:
You mean I'm right about grammar for once 🤣. I can change if you think it will be better for web standardisation, or we can stick to the official British spellings?

Reply:
Doesn't matter to me too much. Set the direction and I'll amend and merge the change

Reply:
We're in Britain, so will remain with British spelling - probably will look better as well on written academic reports.

Reply:
Did you at least apply the grammar fixes if not the spelling ones?

Reply:
But I like using &

Reply:
Going through now...

Reply:
Ok, so I've edited the comment to have just the relevant fixes now

Reply:
Added changes (had to commit  separately due to keeping 'licence')